dev/helm/values.yaml

```
@@ -98,6 +98,22 @@ tolerations: []
 
 affinity: {}
 
+extraEnv:
+  env: []
+#    - name: SAMPLE_ENV
+#      value: "sample value"
+#    - name: SAMPLE_ENV_FROM_SECRET
+#      valueFrom:
+#        secretKeyRef:
+#          name: 'secret-name'
+#          key: 'secret-key'
+  secrets: []
+#    - secret-name
+#    - another-secret-name
+  configmaps: []
```

IMO, this should be camelCase, as it is found in the k8s API *(`configMapRef`)*.

```suggestion
  configMaps: []
```